src/pages/innovation-day/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: 'wasmCloud Innovation Day 2024'
+description: 'Register for wasmCloud Innovation Day 2024'
+---
+
+![wasmCloud Innovation Day](../../../static/pages/home/innovation-day/innovation-day.png)
+
+
+**Time:** Wednesday, September 18, 2024 @ 10am - 6pm Eastern  
+**Place:** Virtual
+
+Whether you’re already using Wasm or a complete newbie, this one-day virtual event is the ideal immersion in all things WebAssembly and wasmCloud. We’ll explore wasmCloud’s technical architecture and growing feature set&mdash;and most importantly, hear how companies in a host of industries are working with this popular open source CNCF project.
+
+Here’s what to expect…check out the full agenda below!
+
+* **wasmCloud in Industry:** Hear how engineers from the world’s largest organizations are using wasmCloud in real-world use cases.
```

Comment:
I would remove the bullets on these 3 points as bolding gives it enough emphasis, I reckon.